packages/hagrid/hagrid/orchestra.py

```diff
@@ -259,6 +262,7 @@ def deploy_to_python(
                 host=host,
```

Comment:
todo: also add queue options here

Reply:
Not sure how we exactly want to organize default setup for docker/k8s? Do we want dynamic consumer creation on as default? Or do we want to have a default set of consumers? Currently async execution is dependent on consumers, so if we have none we cannot execute async work.